alica_ros_proxy/src/clock/AlicaROSClock.cpp

```diff
@@ -1,20 +1,17 @@
 #include <clock/AlicaROSClock.h>
-
 #include "ros/time.h"
 
 namespace alicaRosProxy
 {
 
-AlicaROSClock::AlicaROSClock()
+alica::AlicaTime AlicaROSClock::now() const
 {
-    ros::Time::init();
```

Comment:
is ros::Time::init()    not needed?

Reply:
I think this only used when ros time not initialized by another node. So I think if we add back, it will be helpful in tests or some stuff(where ros time not initialized) else won't be needed. [ref](http://docs.ros.org/diamondback/api/rostime/html/time_8cpp_source.html#l00260)